### ISSUE TITLE
Add a utility method to simplify reading from files

### ISSUE THREAD

**Issue by ennerf** [HebiRobotics/MFL#38]
I keep finding myself writing the same lines of code for loading files all the time.

```Java
try(Source source = Sources.openFile("data.mat"){
    MatFile mat = Mat5.newReader(source).readFile();
}
```

This should become a helper function, e.g., `MatFile mat = Mat5.readFile(File file)`

**Comment by ennerf** [comment]
Added `Mat5.readFromFile` and `Mat5.writeToFile` to latest snapshot

**Comment by ennerf** [comment]
released in `0.5.2`
